### DISCUSSION TITLE
扫描背包物品的方案探讨，以及提高准确率的探讨

### DISCUSSION THREAD

**Discussion by wormtql** [wormtql/yas#21] [General]
## 👋 Yas未来计划支持扫描背包、角色等数据
但是作者并不熟CV，特来探讨可行的深度学习方案，或者传统CV方案  
目前想到的背包识别方案：
  - 与圣遗物一样，检测物品位置，CNN识别物体，CRNN识别数量，但是y轴波动会很大
  - 动态检测物品位置

## 如何提高CRNN准确率，或者其他的识别方案
目前是MobileNetV3_Small+LSTM
- 贴paper链接

**Comment by cdowen** [comment]
作者还有兴趣吗？我正在做动态检测圣遗物框位置，目前是为了让playcover的兼容性更好一点。

另外个人感觉这种识别的目标其实很有限，未必需要深度学习法案，Bag-of-words应该就行。

**Reply by wormtql** [reply]
很欢迎动态框位置检测，bow可以试试，要是效果更好也能考虑
